SU2_CFD/src/hybrid_RANS_LES_model.cpp

```diff
@@ -534,6 +536,15 @@ void CHybrid_Mediator::ComputeInvLengthTensor(CVariable* flow_vars,
     }
   }
 
+  // Make tauSGET trace = 0
+  su2double traceTauSGET = 0;
+  for (iDim =0; iDim < nDim; iDim++) {
+    traceTauSGET += tauSGET[iDim][iDim];
+  }
+  for (iDim = 0; iDim < nDim; iDim++) {
+    tauSGET[iDim][iDim] -= traceTauSGET/3;
+  }
+
```

Comment:
@trevilo Shouldn't we subtract out the trace of tauSGET where it's defined in the CAvgGrad_Hybrid as well?

In the long term, we need to find a way to eliminate the duplication between numerics and the InvLengthTensor calculation.  I've got an idea or two, but I haven't had time to work on them.

Reply:
Oops, I think I missed this one the first time through.

Anyway... yeah, it should be consistent.  I don't really know that trace should be subtracted out in either spot.  In the model, the trace of tauSGET contributes to the modeled stress in the same way as (2/3)*alpha*k, so it at least ought to average to zero to avoid contaminating k.  The easiest way to achieve that is simply to have the trace be zero.  I'm pretty sure it was there at one point in CAvgGrad_Hybrid, b/c I remember commenting it out while investigating stability problems.

Bottom line: I'm fine with including it in CAvgGrad_Hybrid too, assuming stability is ok.